packages/toolpad-app/src/toolpad/propertyControls/GridColumns.tsx

```diff
@@ -190,16 +190,14 @@ function GridColumnsPropEditor({
             <DialogTitle>Edit columns</DialogTitle>
             <DialogContent>
               <Tooltip title="Recreate columns">
-                <IconButton
-                  aria-label="Recreate columns"
-                  onClick={handleRecreateColumns}
-                  disabled={!hasColumnSuggestions}
-                >
-                  <RefreshIcon />
-                </IconButton>
+                <span>
```

Comment:
this prevents it from passing the aria label to the button, right?

Reply:
it's passed to the span, actually that might not be ok good point

Reply:
i added `describeChild` to the tooltip and `aria-label` in the button, seems like the right way to do it